src/deepsparse/utils/time.py

```diff
@@ -49,3 +55,18 @@ def get_new_timer(self):
     def update(self, measurements: Dict[str, float]):
         with self.lock:
             self.measurements.append(measurements)
+
+    def average(self):
+        """Get the average time for each key in every element inside measurements"""
+
+        summary = {"time": defaultdict(float), "execution": defaultdict(int)}
```

Comment:
need a better name for "execution" to make it clear that it's the number of times an operator is executed per inference. also let's add an `average_times` function as well that just returns the "time" dict